byron/ledger/executable-spec/src/Ledger/Update.hs

```diff
@@ -68,7 +68,9 @@ data PParams = PParams -- TODO: this should be a module of @cs-ledger@.
   , _stableAfter :: Core.BlockCount
   -- ^ Chain stability parameter
   , _factorA :: Int
+  -- ^ Minimum fees per transaction
   , _factorB :: Int
```

Comment:
I know it's not part of this PR, but if lovelace have a specific signedness and bit-size, why not a and b ?

Reply:
I don't understand what you mean here :grin: Sorry...

Reply:
I think Lovelace is something like `Word64`, i.e., fixed 64 bit size integral type (without sign), `Int` is a machine specific bit-size. Wouldn't it make sense to also fix `_factorA` / `_factorB` with 64 bits? (In particular as "factorA" is described as "minimum fees per transaction")

Reply:
Oh, in our specs `Lovelace` is an `Integer`, so it is not fixed yet... Although I don't know how @ruhatch handled that when elaborating the abstract transactions to concrete ones ...